README.rst

```diff
@@ -0,0 +1,105 @@
+Flask-SQLAlchemy
+################
+
+.. image:: https://travis-ci.org/pallets/flask-sqlalchemy.svg?branch=2.x-maintenance
```

Comment:
I prefer not to have badges in the Pallets readmes. I find them more distracting than helpful, and the relevant links are at the bottom already.

Reply:
I actually find them really helpful, but I'm happy to defer here.  :)  I'll remove them.